yang/IETF-04/ietf-l3vpn-ntw.yang

```diff
@@ -1406,6 +1406,74 @@ module ietf-l3vpn-ntw {
                            diagnosis purposes. The semantic of the description
                            is local to an implementation.";
                       }
+                      leaf as-override {
+                        type boolean;
+                          default false;
+                          description
+                           "Define whether AS override is enabled, i.e., replace
+                           the AS number of the peer specified in the AS Path
+                           attribute with the local AS number.";
+                        }
+
+                      leaf default-route {
+                        type boolean;
+                        default false;
+                        description
+                         "Define whether default route can be advertised
+                         to its peer.";
+                        }
+
+
+                      container bgp-max-prefix {
```

Comment:
Add description. For example
"Maximimum number of BGP prefixes allowed in the BGP session. Allows to control how many prefixes can be received from a neighbor. If the limit is exceeded, the session will be teared down."
Why 5000? Any reference to support it?
Any reference to the parameter? RFC4271 ?